bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+
+        <tr>
+        
+
+        <tr>
+            <td><code>profv:mostSignificantWord</code></td>
+            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping). If the term is not provided, the TD processor defaults it to <code>true</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+            
+
+        <tr>
+
+        <tr>
+            <td><code>profv:method</code></td>
+            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping). If the term is not provided, the TD processor defaults it to <code>true</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#string">string</a><p>(one of "READ", "WRITE")</p></td>
+            
+
+        <tr>
+        
+
+        <tr>
+            <td><code>profv:pollingTime</code></td>
+            <td>IO supervisor polling rate in milliseconds. It is recommended to keep the polling rate above IO device send clock time. this will ensure that the cyclic communication will not be interferrupted in anyway.</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+            
+
+        <tr>
+        
+        <tr>
+            <td><code>profv:timeout</code></td>
+            <td>PROFINET response maximum waiting time. Defines how much time the runtime should wait until it receives a reply from the device.</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+        
+
+        <tr>
+
+        <tr>
+            <td><code>profv:offset</code></td>
+            <td>used for <code>object</code> type properties to identify the starting point within a byte stream payload that represent a property.</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+        
+
+        <tr>
+            <td><code>profv:byte</code></td>
+            <td>used for <code>object</code> type properties to identify the byte length of a proterty from the byte stream payload</td>
```

Comment:
proterty  --> property ?

Later there is proterty  again

Reply:
```suggestion
            <td>For <code>object</code> type properties. Used to identify the byte length of a property from the byte stream payload</td>
```